lib/db_connection/connection_pool.ex

```diff
@@ -149,6 +150,14 @@ defmodule DBConnection.ConnectionPool do
     drop_idle(past_in_native, status, queue, codel)
   end
 
+  def handle_info(
```

Comment:
This is the part that I'm concert about. Too many `:checkout_timeout`s could lead to the ConnectionPool being overflowed with messages, unable to handle anything else.

Any thoughts on this?